client/my-sites/stats/geochart/index.jsx

```diff
@@ -142,7 +142,7 @@ class StatsGeochart extends Component {
 		return (
 			<div>
 				<div ref="chart" className={ classes } />
-				{ siteId && <QuerySiteStats statType={ statType } siteId={ siteId } query={ query } /> }
+				{ siteId && <QuerySiteStats statType={ statType } siteId={ siteId } query={ query } heartbeat={ 60000 } /> }
```

Comment:
As I mentioned during the hangout, maybe we can bump this up to 2 or 3 minutes.  Eventually we might consider disabling the heartbeat on stat periods that are not "active", as-in the data will not change if the period does not include today.